src/native/eventpipe/ds-process-protocol.h

```diff
@@ -58,6 +58,56 @@ ds_process_info_payload_init (
 void
 ds_process_info_payload_fini (DiagnosticsProcessInfoPayload *payload);
 
+/*
+* DiagnosticsProcessInfo2Payload
+*/
+
+// command = 0x0400
+#if defined(DS_INLINE_GETTER_SETTER) || defined(DS_IMPL_PROCESS_PROTOCOL_GETTER_SETTER)
+struct _DiagnosticsProcessInfo2Payload {
+#else
+struct _DiagnosticsProcessInfo2Payload_Internal {
+#endif
+	// The protocol buffer is defined as:
+	// X, Y, Z means encode bytes for X followed by bytes for Y followed by bytes for Z
+	// uint = 4 little endian bytes
+	// long = 8 little endian bytes
+	// GUID = 16 little endian bytes
+	// wchar = 2 little endian bytes, UTF16 encoding
+	// array<T> = uint length, length # of Ts
+	// string = (array<char> where the last char must = 0) or (length = 0)
+
+	// ProcessInfo = long pid, string cmdline, string OS, string arch, GUID runtimeCookie, string managed entrypoint assembly path, string clr product version
```

Comment:
Nit: comment order of fields doesn't match the protocol

Reply:
```suggestion
	// ProcessInfo = long pid, GUID runtimeCookie, string cmdline, string OS, string arch, string managed entrypoint assembly path, string clr product version
```